src/test/java/org/broadinstitute/consent/http/service/DarCollectionServiceTest.java

```diff
@@ -312,6 +308,55 @@ void testCreateElectionsForDarCollection() throws Exception {
     verify(darCollectionDAO, times(1)).findDARCollectionByCollectionId(any());
   }
 
+  @Test
+  void testCreateElectionsForDarCollectionEmpty() {
+    User user = new User();
+    user.setEmail("email");
+    DataAccessRequest dar = new DataAccessRequest();
+    dar.setReferenceId(UUID.randomUUID().toString());
+    DarCollection collection = createMockCollections(1).get(0);
+    collection.setDars(Map.of(dar.getReferenceId(), dar));
+
+    assertThrows(IllegalStateException.class, () -> {
+      service.createElectionsForDarCollection(user, collection);
+    });
+  }
+
+  @Test
+  void testCreateElectionsForDarCollectionNull() throws Exception {
+    User user = new User();
+    user.setEmail("email");
+    DataAccessRequest dar = new DataAccessRequest();
+    dar.setReferenceId(UUID.randomUUID().toString());
+    DarCollection collection = createMockCollections(1).get(0);
+    collection.setDars(Map.of(dar.getReferenceId(), dar));
+    when(darCollectionServiceDAO.createElectionsForDarCollection(any(), any())).thenReturn(null);
```

Comment:
I believe this is the same as not mocking at all, since all mocked APIs return `null` by default. If you only want to verify that an API is called (which mocking will do if you have strict stubbing enabled), I would instead use `verify` after the API is called, to make the intent of the code clearer.

Reply:
It seems to return an empty list by default, hence the separate test for `null`.

Reply:
You're right, I didn't know that about mockito. That makes sense though since I don't think collection APIs should ever return nulls in the first place..